ceph-salt.spec

```diff
@@ -94,13 +94,18 @@ mkdir -p %{buildroot}%{fdir}/metadata/%{fname}/
 cp -R ceph-salt-formula/salt/* %{buildroot}%{fdir}/states/
 cp ceph-salt-formula/metadata/* %{buildroot}%{fdir}/metadata/%{fname}/
 
+# manpage
+mkdir -p %{buildroot}%{_mandir}/man8/
+cp ceph-salt.8 %{buildroot}%{_mandir}/man8/ceph-salt.8
+
 
 %files
 %license LICENSE
 %doc CHANGELOG.md README.md
 %{python3_sitelib}/ceph_salt*/
 %{_bindir}/%{name}
 %dir %{_datadir}/%{name}
+%{_mandir}/man8/ceph-salt.8*
```

Comment:
please insert `%doc ` at the beginning of this line

Reply:
it might also be worth considering to either move the line up so it's adjacent to the other `%doc` line, or even merging the two `%doc` lines into one

Reply:
On second thought, no, don't merge them into one. This line

```
%doc CHANGELOG.md README.md
```

causes RPM to create a documentation directory for ceph-salt and copy the files into it, whereas this line:

```
%doc %{_mandir}/man8/ceph-salt.8
```

causes RPM to install the file in `%_mandir`.

By the way, looking at e.g. `rpm -ql man-pages`, it seems like the man page should be gzipped (and only the gzipped version should be the RPM)